content/chainguard/vms/compliance-features.md

```diff
@@ -0,0 +1,97 @@
+| **CIS benchmark compliance**        | [CIS](https://www.cisecurity.org/cis-benchmarks/cis-benchmarks-faq) Level 1 hardened variants; hybrid STIG \+ CIS baseline. |
+| **Secure Boot**                     | Secure Boot enabled by default across AWS, Azure, GCP, and on-prem. |
+| **Compliance evidence & reporting** | FIPS certificates, OpenSSL docs, SCAP scan results, and POA\&M-ready artifacts. |
+| **CVE remediation SLA**             | 7 days for critical CVEs, 14 days for high, medium, and low. |
+
+Chainguard FIPS 140-3 validated and hardened VM images serve as ready-to-use replacements for standard operating systems across AWS, Azure, and GCP, allowing organizations to maintain existing infrastructure and workflows while achieving immediate compliance. Read on this doc to learn more about Chainguard VMs compliance features and how they can help reduce engineering toil from your organization.
+
+## **FIPS 140-3 Validated Cryptography**
+
+Chainguard VMs include FIPS 140-3 validated software cryptographic modules, backed by a NIST Cryptographic Module Validation Program (CMVP) certificate.
+
+* **Validated modules**
+    * Cryptographic modules are validated under FIPS 140-3 and integrated directly into the base image.
+    * The images contain SP 800-90B compliant entropy sources for strong, standards-aligned randomness.
+* **Runtime guardrails**
+    * Guardrails prevent the use of non-FIPS-approved cryptography at runtime (e.g., enforcing FIPS-approved ciphers and modes only).
+* **Documentation for auditors**
+    * FIPS integration documentation, including OpenSSL certificates and details on module configuration, is shipped as part of the deliverables to streamline audit review and ATO packages.
+
+This setup allows teams to consume an OS image that is already FIPS-conformant at the platform layer rather than building and validating crypto modules in-house.
+
+## **STIG Hardening**
+
+Chainguard VMs provide variants hardened to DISA Security Technical Implementation Guide (STIG) requirements used across U.S. federal and defense environments.
+
+* 2000+ relevant system controls are pre-configured and validated in the image, not delivered as a checklist that the customer must implement.
+* Images are designed as ready-to-use replacements for existing Linux OS images, enabling STIG-compliant baselines without re-architecting existing infrastructure.
+
+Chainguard can also provide SCAP scan outputs aligned with STIG requirements, helping teams demonstrate compliance with control requirements during audits.
+
+## **CIS Benchmark Compliance**
+
+For organizations standardizing on CIS controls, Chainguard offers images hardened to **CIS Level 1** benchmarks.
+
+* Images are available in **CIS L1 hardened variants** for Chainguard VMs.
+* Chainguard VMs use a **hybrid baseline combining CIS L1 benchmarks with STIG requirements and industry-recognized secure defaults** to provide defense-in-depth hardening.
```

Comment:
```suggestion
For organizations standardizing on CIS controls, Chainguard offers images hardened to **CIS Level 1** benchmarks. Chainguard VMs use a hybrid baseline combining CIS L1 benchmarks with STIG requirements and industry-recognized secure defaults to provide defense-in-depth hardening.
```

I feel if that much of the sentence is bolded it ends up looking off rather than drawing attention to anything. Also I felt like the first bullet was just repeating the first sentence.